### DISCUSSION TITLE
apache skywalkinfg configuration for ECS fargate

### DISCUSSION THREAD

**Discussion by ssvssv** [apache/skywalking#11687] [Q&A]
How can I configure apache skywalking to get ECS fargate logs? I didn't find documentation for configuring this APM for ECS fargate!

**Comment by heyanlong** [comment]
I thank you can observe ECS the same way as observe S3
https://skywalking.apache.org/blog/2023-03-12-skywalking-aws-s3-eks/
https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-aws-eks-monitoring/

**Reply by heyanlong** [reply]
Because they all use [Amazon CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)

**Reply by ssvssv** [reply]
Let me try. Thank you in advance.

**Comment by wu-sheng** [comment]
For AWS services, there are two ways,
1. Through standard otel integration, like EKS monitoring.

https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-aws-eks-monitoring/

2. If the service doesn't have, you could refer how s3 monitoring works, mentioned above.

https://skywalking.apache.org/docs/main/next/en/setup/backend/aws-firehose-receiver/

**Reply by ssvssv** [reply]
Let me try. Thank you for immediate response.
